src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -141,15 +146,17 @@ export const MarketplaceSelect = () => {
           </Box>
         </Box>
         <Column className={styles.filterDropDowns} paddingBottom="8" paddingLeft="0">
-          {Object.entries(marketPlaceItems).map(([value, title]) => (
-            <MarketplaceItem
-              key={value}
-              title={title}
-              value={value}
-              count={marketCount?.[value] || 0}
-              {...{ addMarket, removeMarket, isMarketSelected: selectedMarkets.includes(value) }}
-            />
-          ))}
+          {Object.entries(marketPlaceItems)
+            .filter(([value]) => isNftGraphQl || !graphQlMarkets.includes(value))
```

Comment:
it could be better to `useMemo` this so the filtering isn't happening every render if the conditions aren't changing.